src/test/java/alinaTinca/tests/Tema5/Demoqa/Elements/Buttons.java

```diff
@@ -0,0 +1,40 @@
+package alinaTinca.tests.Tema5.Demoqa.Elements;
+
+import AlinaTinca.BrowserManager;
+import AlinaTinca.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.List;
+
+
+public class Buttons {
+
+    public static void main(String[] args) {
+        goToTheButtons();
+        //check if the method called goToTheButtons is done successfully
+
+    }
+
+    public static void goToTheButtons() {
+        //create a method called goToTheButtons that goes to the Buttons sections and interacts with a button
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        //create and configure the Chrome browser page setting
+        driver.get("https://demoqa.com/buttons");
+        //go to the Demoqa URL
+        WebElement buttonsSection = driver.findElement(By.xpath("//div[@class='element-list collapse show']/ul/li[5]"));
+        //find the Buttons section
+        buttonsSection.click();
+        //click on this section
+
+        WebElement clickMeButton = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[2]/div[2]/div[3]/button"));
```

Comment:
You ca use the following relative xpath for this button -> //button[text()='Click Me']